website/src/components/Sidebar.vue

```diff
@@ -77,13 +89,55 @@ export default {
   }
 }
 
+.item-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+.expand {
+  width: 35px;
+  border: 0;
+  outline: none;
+  background: transparent;
+  cursor: pointer;
+  & > span {
+    position: relative;
+    display: block;
+    &:before, &:after {
```

Comment:
line breaks should be inserted before `& ...`

Reply:
?